src/ansys/dpf/core/server.py

```diff
@@ -355,6 +359,10 @@ def connect():
         dpf.core._server_instances.append(weakref.ref(server))
         return server
 
+    # Enforce LegacyGrpc when on macOS
+    if platform.system() == 'Darwin':
+        config = dpf.core.AvailableServerConfigs.LegacyGrpcServer
```

Comment:
@PProfizi maybe we could add mac agents to the CI/CD (at least for the current server version) to be able to test all this. I don't know how difficult would this be, as all the current testing matrices that we have tend to run things for CLayer InProcess, CLayer gRPC and LegacyGrpc, but it could be worth doing if we start t make these kind of distinctions